rskj-core/src/test/java/org/ethereum/config/blockchain/upgrades/ActivationConfigsForTest.java

```diff
@@ -36,15 +42,15 @@ private static List<ConsensusRule> getPaidBridgeTxsRskip() {
 
     private static List<ConsensusRule> getOrchidRskips() {
         return new ArrayList<>(Arrays.asList(
-            ConsensusRule.RSKIP85,
-            ConsensusRule.RSKIP87,
-            ConsensusRule.RSKIP88,
-            ConsensusRule.RSKIP89,
-            ConsensusRule.RSKIP90,
-            ConsensusRule.RSKIP91,
-            ConsensusRule.RSKIP92,
-            ConsensusRule.RSKIP97,
-            ConsensusRule.RSKIP98
+                ConsensusRule.RSKIP85,
+                ConsensusRule.RSKIP87,
+                ConsensusRule.RSKIP88,
+                ConsensusRule.RSKIP89,
+                ConsensusRule.RSKIP90,
+                ConsensusRule.RSKIP91,
+                ConsensusRule.RSKIP92,
+                ConsensusRule.RSKIP97,
+                ConsensusRule.RSKIP98
```

Comment:
Can we undo all these changes? I think we have consensus to use one tab (4 spaces) for indentation